packages/backend/tests/commands-a.spec.ts

```diff
@@ -61,6 +81,31 @@ describe("commands unit tests", () => {
     });
   });
 
+  describe("onErrorCfLogin", () => {
+    it("should handle login error by calling cmdLogin and retrying", async () => {
+      const errorMessage = "test error message";
+      const error = new Error(errorMessage);
+
+      try {
+        await internal.onErrorCfLogin(error, "https://example.com");
+      } catch (err) {
+        console.log(err);
```

Comment:
consider to remove - redundant/confusing output

Reply:
@alex-gilin fixed